micromasters/settings.py

```diff
@@ -557,6 +557,17 @@ def get_all_config_keys():
     in get_all_config_keys() if key.startswith(MM_FEATURES_PREFIX)
 }
 
+MIDDLEWARE_FEATURE_FLAG_QS_PREFIX = get_var("MIDDLEWARE_FEATURE_FLAG_QS_PREFIX", None)
+MIDDLEWARE_FEATURE_FLAG_COOKIE_NAME = get_var('MIDDLEWARE_FEATURE_FLAG_COOKIE_NAME', 'MM_FEATURE_FLAGS')
+MIDDLEWARE_FEATURE_FLAG_COOKIE_MAX_AGE_SECONDS = get_var('MIDDLEWARE_FEATURE_FLAG_COOKIE_MAX_AGE_SECONDS', 60 * 60)
+
+
+if MIDDLEWARE_FEATURE_FLAG_QS_PREFIX:
```

Comment:
The possible values are either `None` or `str` (please see usage [here](https://github.com/mitodl/micromasters/pull/2568/files#diff-910e7ef6baffbc57d4ad11ad102ceb5aR26)). I coded it this way so that the feature is only enabled if the prefix has a non-empty value, that gives us two behaviors I think are desirable a) the prefix needs to be set to something other than `None` or `""`, this means the feature will never be enabled with no prefix b) it reduces configuration complexity by eliminating the need to set both the prefex environment variable **and** a boolean variable to enable the middleware. Disabling the middleware is then as simple as either setting the env var to `""` or unsetting it.